recipes/bonjour/system/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class BonjourSystemConan(ConanFile):
+    name = "bonjour"
+    provides = "mdnsresponder"
+    version = "system"
+    description = "Conan package for Apple's Bonjour"
+    topics = ("Bonjour", "DNS-SD", "mDNS")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://developer.apple.com/bonjour/"
+    license = "Apache-2.0", "BSD-3-Clause"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def validate(self):
+        if self.settings.os != "Windows" or (hasattr(self, "settings_build") and tools.cross_building(self, skip_x64_x86=True)):
+            raise ConanInvalidConfiguration("Only Windows is supported for this package.")
+
+    def system_requirements(self):
+        # extract and install Bonjour64.msi from downloaded BonjourPSSetup.exe
+        # see https://community.chocolatey.org/packages/bonjour#files
+        package_tool = tools.SystemPackageTool(tool=tools.ChocolateyTool())
+        package_tool.install(update=True, packages="bonjour")
```

Comment:
Not sure to understand, source code is available at https://opensource.apple.com/tarballs/mDNSResponder/, this recipe should compile it. system recipes are evil, we try to keep them to the bare minimum.

Reply:
The source code for Windows is not available in recent packages. And the Visual Studio projects are massively out of date in the earlier ones and the Makefile does not support Windows. See discussion in #6930.

Reply:
How is it build by chocolatery?

Makefike is not an issue, we can fix build files, eventually write a CMakeLists from scratch.

Reply:
As I said, it's _not_ built by Chocolatey, it installs using the MSI extracted from an installer downloaded from Apple. And they don't install the necessary header file and stub lib.

AFAIK, the mDNSPosix source code used by the Makefile was never intended to work on Windows, that's why there was mDNSWindows code in earlier versions of mDNSResponder.

Reply:
@SpaceIm, FYI, #6930 now attempts to build the last available Windows version of mDNSResponder on Windows. I do have the same concerns about not using the installer as raised in https://github.com/conan-io/conan-center-index/pull/6928#discussion_r693353429 about the Avahi system package. It seems to me that Conan is perhaps not yet suited to building and deploying software that includes daemons/services which you shouldn't run multiple instances of.